app/api/orders.py

```diff
@@ -40,6 +40,16 @@
 alipay_blueprint = Blueprint('alipay_blueprint', __name__, url_prefix='/v1/alipay')
 
 
+def check_billing_info(data):
+    if data.get('payment_mode') in ['stripe', 'paypal', 'alipay', 'omise'] and not data.get('is_billing_enabled'):
+        raise ConflictException({'pointer': '/data/attributes/is_billing_enabled'},
```

Comment:
How is this a Conflict? This is 400 Bad Request

Reply:
@iamareebjamal  Currently, we don't have BadRequestException with 400 status in `helpers/exceptions.py`
Should I add it or use `UnprocessableEntity` instead? However, 422 status is responded from the server if we use UnprocessableEntity which states that syntax is correct.

Reply:
Yes, I meant to say 422